ui/src/helpers/get-status-colors.ts

```diff
@@ -40,6 +42,13 @@ export function getStatusBackgroundColor(status: Status): string {
   return STATUS_BACKGROUND_COLOR[status]; // This will now only be called for defined statuses
 }
 
+// Get the color class for coloring the background of vulnerability ratings
+export function getVulnerabilityRatingBackgroundColor(
+  rating: VulnerabilityRating
+): string {
+  return VULNERABILITY_RATING_BG_COLOR[rating];
+}
```

Comment:
Do we need a function, or would just using `VULNERABILITY_RATING_BG_COLOR[rating]` work?

Reply:
The implementation just mirrors how it was done earlier, since for the other functions, `undefined` needed its own handling, therefore they had to be functions. But I can shortcut this.

Reply:
As a small cleanup at some point, it would be good to move the constants like `VULNERABILITY_RATING_BG_COLOR` to this file and not export them so that there's only one way to get the colors.